src/views/pages/index.html

```diff
@@ -0,0 +1,9 @@
+<%~ include('partials/head.html') %>
```

Comment:
You can use any templating engine to include partials, e.g.  very fas `Eta` (ESJ like).